packages/termux-services/build.sh

```diff
@@ -0,0 +1,8 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/Neo-Oli/termux-services/
```

Comment:
Would be better if those few shell scripts were stored in this Termux repo instead.

Reply:
I started it like this, but as I added more scripts I found it easier just to write a short Makefile. I would definitely suggest recreating this Repository at https://github.com/termux/termux-services/ instead of my personal space.